### ISSUE TITLE
Veteran Status - App Store Image Update - Updating Home Page to show Proof of Veteran Status 

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#6770]
**Purpose:** 
Part of the go to market strategy requires the Apple, Google, and VA app stores to be updated if needed. Veteran status modifies the name tag on both the home page and profile thus need to update those images according. 

**Other info:**
- [Updating app store documentation ](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Operations/Updating%20the%20App%20Stores/)
- Google Play store only shows mobile images
- Va.gov app page only shows mobile images
- Apple store is the only one that shows both mobile and tablet 
- Jen recommended sizing this as a 3
- Planned for this sprint but lost the ticket so creating a new one 
- [Summary page content ](https://www.figma.com/file/UOTRHWoB1eNZE0M3P16Su2/%F0%9F%A7%B0-App-Store-Images---Resource---VAMobile%F0%9F%A7%B0?type=design&node-id=677-2264&mode=design&t=2QmEVtwzDO9zG8S9-0)

**ACs:**

- [x] - Create image for [Apple Store](https://apps.apple.com/us/app/va-health-and-benefits/id1559609596?platform=iphone)
   - [x] iPad - summary page   
    - [x] iPad - home page
    - [x] iPad - profile
   - [x] iPhone - summary page   
    - [x] iPhone - home page
    - [x] iPhone- profile
- [x] - Create image for [Google Play Store](https://play.google.com/store/apps/details?id=gov.va.mobileapp&hl=en_US&gl=US)
    - [x] Summary page
    - [x] Home Page mobile 
    - [x] Profile page mobile
- [x] - Create image for [VA.gov app page ](https://mobile.va.gov/app/va-health-and-benefits)
    - [x] Summary page
    - [x] Home Page mobile 
    - [x] Profile page mobile

**Comment by bischoffa** [comment]
[Slack convo ](https://dsva.slack.com/archives/C018V2JCWRJ/p1695147068902709)of decision to include summary page

**Comment by bischoffa** [comment]
According to the [documentation](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Operations/Updating%20the%20App%20Stores/), it does require engineer work to take the screenshots 

Binny confirmed he can get them

**Comment by brea11y** [comment]
@bischoffa – The updated app store images have been added to the [Google drive folder](https://drive.google.com/drive/folders/1EW_2Vh2gP7rY6-kUN4dsVrui6YN9UTJa?usp=drive_link) (per the documentation instructions). I wasn't sure if we would only need the provide the app store with the ones we wanted to update (summary, home, and profile) or all of them, so I just added all of them into the folder(s). I believe the next step is for you to review them. Since this ticket was only related to the design work, I'll close it out with this comment.
